code/modules/station_goals/vault_mutation.dm

```diff
@@ -1,125 +1,125 @@
-/datum/mutation/human/breathless
+/datum/mutation/human/Breathless
 	name = "Breathless"
 	desc = "A mutation within the skin that allows for filtering and absorption of oxygen from the skin."
 	text_gain_indication = "Your lungs feel great."
 	text_lose_indication = "Your lungs feel normal again."
 	locked = TRUE
 	mutadone_proof = TRUE
 
-/datum/mutation/human/breathless/on_acquiring(mob/living/carbon/human/acquirer)
+/datum/mutation/human/Breathless/on_acquiring(mob/living/carbon/human/acquirer)
 	. = ..()
-	ADD_TRAIT(acquirer, TRAIT_NOBREATH)
+	ADD_TRAIT(acquirer, TRAIT_NOBREATH, GENETIC_MUTATION)
 
-/datum/mutation/human/breathless/on_losing(mob/living/carbon/human/owner)//this shouldnt happen under normal condition but just to be sure
+/datum/mutation/human/Breathless/on_losing(mob/living/carbon/human/owner)//this shouldnt happen under normal condition but just to be sure
 	. = ..()
-	REMOVE_TRAIT(owner, TRAIT_NOBREATH)
+	REMOVE_TRAIT(owner, TRAIT_NOBREATH, GENETIC_MUTATION)
 
-/datum/mutation/human/fast_runner
+/datum/mutation/human/Fast_Runner
```

Comment:
NO NOT IN THE PATH

Reply:
IN THE NAME

Reply:
you didnt fix this

Reply:
what do you mean?

Reply:
i thought i fixed that in the earlier commit, its fixed now.

Reply:
you didnt fix it you just changed the name